XCUITests/FxScreenGraph.swift

```diff
@@ -242,77 +284,94 @@ func createScreenGraph(_ app: XCUIApplication, url: String = "https://www.mozill
     map.createScene(OpenWithSettings) { scene in
         scene.backAction = navigationControllerBackAction
     }
+    
+    map.createScene(ShowTourInSettings) { scene in
+        scene.gesture(to: Intro_FxASignin) {
+            let introScrollView = app.scrollViews["IntroViewController.scrollView"]
+            for _ in 1...4 {
+                introScrollView.swipeLeft();
+            }
+            app.buttons["Sign in to Firefox"].tap()
+        }
+        scene.backAction = {
+            introScrollView.swipeLeft()
+            startBrowsingButton.tap()
+        }
+        
+    }
+    
+    map.createScene(Intro_FxASignin) { scene in
+        // Note: if a browser tab was open beforehand, this will go back to
+        // BrowserTab. In such case, manual redirection with nowAt() is needed
+        scene.tap(app.navigationBars["Client.FxAContentView"].buttons["Done"], to: HomePanelsScreen)
```

Comment:
Please add `accessibilityIdentifier` to any views that need them.

Reply:
Hmm, actully "Client.FxAContentView" is acc.id, of the header view, and since they're reusing them, I'm not sure how to add acc.id to the buttons themselves.  I'll look a bit more into this.